### PR TITLE
Added registration, FAQ, and schedule, and Google Analytics.

### PR DESCRIPTION
**Pull request by bezi**

### CODE REVIEW

**Comment by tomshen** [comment]
Why did you make a new PR? Any commits you make would've been added to the last PR.

**Comment by bezi** [comment]
This was news to me.  I thought pull requests would be tied to commits.

File: index.html [hunk 1]
```diff
@@ -18,13 +18,152 @@
             <div id="splash-info">
                 <img class="logo" src="assets/imgs/logo.png" alt="TartanHacks logo"/>
                 <h1>TartanHacks</h1>
-                <p>TartanHacks is an annual, student-run hackathon run by <a href="http://www.scottylabs.org/">Scottylabs</a>.</p>
+                <p>TartanHacks is an annual hackathon run by <a href="http://www.scottylabs.org/">Scottylabs</a>, a student organisation.</p>
```
Comment (justingallagher) [inline index.html]: You might want to mention that this is at CMU

Comment (Z1MM32M4N) [inline index.html]: I don't mean to nitpick this to death, but I think it sounds a little better to say

"... a student organization __at CMU__."

**Comment by tomshen** [comment]
@bezi Nope, they're [tied to branches](https://help.github.com/articles/using-pull-requests/#sending-the-pull-request):

> After your pull request is sent, any new commits pushed to your branch will automatically be added to the pull request. This is especially useful if you need to make more changes.

**Comment by jez** [comment]
@bezi Looks like you have one small, cosmetic change to make before we can pull this.

**Comment by bezi** [comment]
What change?  I don't think it's necessary to say this is at CMU given our target audience.  But seriously we need to deploy this.

**Comment by tomshen** [comment]
It's good for SEO if nothing else. Just say something about it being CMU's biggest hackathon or something. And our target audience does include people from other schools.

**Pull request merged**